src/admin-events/admin-events.spec.js

```diff
@@ -2,13 +2,16 @@ const ethers = require('ethers');
 const { getAddress } = require('@uma/contracts-node');
 const { createTransactionEvent } = require('forta-agent');
 
-const { createAlert, handleTransaction } = require('./admin-events');
+const { createAlert, handleTransaction, initialize } = require('./admin-events');
 
 const CHAIN_ID = 1;
 const votingAddressPromise = getAddress('Voting', CHAIN_ID);
 
 // Tests
 describe('admin event monitoring', () => {
+  // Initialize the Handler
+  initialize();
```

Comment:
```suggestion
describe('admin event monitoring', async () => {
  // Initialize the Handler
  await initialize();
```

Reply:
Fixed.  I had to move initialize to the it functions since describe cannot be async.